ci/gpu/build.sh

```diff
@@ -220,8 +220,9 @@ gpuci_logger "Test notebooks"
 ${WORKSPACE}/ci/gpu/test-notebooks.sh 2>&1 | tee nbtest.log
 python ${WORKSPACE}/ci/utils/nbtestlog2junitxml.py nbtest.log
 
+return ${EXITCODE}
```

Comment:
Is this an intended change?

Reply:
This change already exists in branch-0.19 -- is GitHub just derping out here?

https://github.com/rapidsai/cudf/blob/branch-0.19/ci/gpu/build.sh#L227

Reply:
Doesn't look like it and this change needs to be reversed

Reply:
@kkraus14 I reversed this change on Friday, awaiting your approval